ci/jenkins/test-mc.sh

```diff
@@ -401,6 +402,7 @@ function run_multicluster_e2e {
     # avoid pulling the image again when running Multi-cluster e2e tests.
     docker pull "registry.k8s.io/e2e-test-images/agnhost:2.29"
     docker save "registry.k8s.io/e2e-test-images/agnhost:2.29" -o "${WORKDIR}"/agnhost.tar
+                ssh -o StrictHostKeyChecking=no -n Administrator@${IP} "rm -rf antrea && mkdir antrea && cd antrea && tar -xzf ../antrea_repo.tar.gz > /dev/null && NO_PULL=${NO_PULL}; DOCKER_NETWORK=host make build-windows && docker save -o antrea-windows.tar antrea/antrea-windows:latest && gzip -f antrea-windows.tar" || true
```

Comment:
This line looks strange, it should align with line 404.
Question: why is this added?